src/api/app/models/relationship.rb

```diff
@@ -108,42 +106,46 @@ def self.add_group(obj, group, role, ignoreLock = nil, check = nil)
     r.delete if r.invalid?
   end
 
-  # this is to speed up secure Project.find
+  # calculate and cache forbidden_project_ids for users
   def self.forbidden_project_ids
-    if User.current
-      return User.current.forbidden_project_ids
-    end
-    # mainly for scripts
-    forbidden_project_ids_for_user(nil)
-  end
+    # Admins don't have forbidden projects
+    return [0] if User.current && User.current.is_admin?
```

Comment:
Before was a comment and a portion of code returning always a 0 inside the array if nothing was inside... we just let the same behavior, but maybe if we check who is using it we can get rid of it